testutil/random.go

```diff
@@ -59,7 +60,8 @@ func RandomCorePubKey(t *testing.T) core.PubKey {
 // RandomEth2PubKey returns a random eth2 phase0 bls pubkey.
 func RandomEth2PubKey(t *testing.T) eth2p0.BLSPubKey {
 	t.Helper()
-	pubkey, _, err := tbls.Keygen()
+	random := rand.New(rand.NewSource(rand.Int63()))
```

Comment:
is this deterministic?

Reply:
yup, since it uses math.Rand for the seed